ytfzf

```diff
@@ -783,13 +805,24 @@ done
 shift $((OPTIND-1))
 #}}}
 
+# Get search{{{
+search="$*"
+get_search () {
+	case "$search" in
+		-) print_info "Reading stdin\n"; read -r search ;;
+		'') search_prompt_menu ;;
```

Comment:
What does $input.do here

Reply:
oops, that line was meant to be removed since it's now redundant to the line `search="$*"`.

fixed.